### PR TITLE
refactor rest api

### PR DESCRIPTION
**Pull request by YoohaBae**
- the directories of the rest api were changed and code was refactored accordingly
- /app/gausstalk/v1/docs shows the api documentation

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,48 +1,23 @@
...
+from fastapi.middleware.cors import CORSMiddleware
 
+from api.v1.api import API
 
-class ConnectionManager:
-    def __init__(self):
-        self.connections: List[WebSocket] = []
+API.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
```
[the remark this answers is not in the document]
Comment (yonsweng) [reply main.py]: I specified the origins. Check out please.

File: api/v1/endpoints/websocket.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    def disconnect(self, websocket: WebSocket):
+        self.connections.remove(websocket)
+
+    async def broadcast(self, data: str):
+        for connection in self.connections:
+            await connection.send_text(data)
+
+
+manager = ConnectionManager()
+Router = APIRouter()
```
[the remark this answers is not in the document]
Comment (yonsweng) [reply api/v1/endpoints/websocket.py]: I made the change. Give some suggestion of this, please.

**Comment by yonsweng** [comment]
@YoohaBae I made some changes. Please review.

File: api/v1/endpoints/websocket.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (YoohaBae) [reply api/v1/endpoints/websocket.py]: I think the changes are good!

**Pull request merged**